src/vmod_queryfilter.c

```diff
@@ -2,6 +2,7 @@
  * libvmod-queryfilter: Simple VMOD for filtering/sorting query strings
  *
  * Copyright 2016 The New York Times Company
+ * Modifications copyright 2017 Norsk Rikskringkasting AS
```

Comment:
@knan-nrk PR looks great! Thank you!

I just have to get approval on formatting regarding the additional copyright before I can merge - it _may_ be preferred to have this [relocated to a NOTICE file](http://www.apache.org/legal/src-headers.html#header-existingcopyright), as ASF does (will verify and get back to you). I don't anticipate complications. :)

Can you advise regarding whether NRK policy is to:
- transfer copyright of the submission to the New York Times company and be listed as a contributor a top-level NOTICE file (e.g., like **[this](https://github.com/apache/httpd/blob/trunk/NOTICE#L7-L9)**)
- maintain copyright ownership of the submitted modifications (displayed in either NOTICE or source file)

_Thanks again for contributing!_

Reply:
@andrew-canaday I've talked to people inhouse, and we have no problem with alternative 1 - transfer copyright and be listed as a contributor.

Thanks for writing the vmod in the first place!